contracts/LifCrowdsale.sol

```diff
@@ -238,7 +238,7 @@ contract LifCrowdsale is Ownable, Pausable {
 
     // if the minimiun cap for the market maker is not reached transfer all funds to foundation
     // else if the min cap for the market maker is reached, create it and send the remaining funds
-    if (this.balance < foundationBalanceCapWei) {
+    if (this.balance <= foundationBalanceCapWei) {
```

Comment:
If its equal it will create the MM with 0 balance

Reply:
Exactly, that's what was happening, we don't want that right? We want to transfer everything up to and including `foundationBalanceCapWei` to the foundation

Reply:
Ho yes thats the other condition, cool.